pyproject.toml

```
@@ -26,6 +26,7 @@ classifiers = [
     'Programming Language :: Python :: 3.10',
     'Programming Language :: Python :: 3.11',
     'Programming Language :: Python :: 3.12',
+    'Programming Language :: Python :: 3.13',
```

You'll need to include this in the pipeline.
```suggestion
    'Programming Language :: Python :: 3.13',
```

---

@Kludex What do you mean include in pipeline? 3.13 already in CI list: https://github.com/Kludex/python-multipart/blob/3e909f52a774365b8817a52d4312db4ba8061d07/.github/workflows/main.yml#L14

---

Ah, nice!